include/i2s.h

```diff
@@ -383,7 +405,10 @@ static inline int i2s_configure(struct device *dev, enum i2s_dir dir,
  * @retval -EBUSY Returned without waiting.
  * @retval -EAGAIN Waiting period timed out.
  */
-static inline int i2s_read(struct device *dev, void **mem_block, size_t *size)
+__syscall int i2s_read(struct device *dev, void **mem_block, size_t *size);
```

Comment:
shouldn't be `void *mem_block` since syscall will copy to this buffer ?

Reply:
You're right -- I need to adjust this a bit.